docs/pages/versions/unversioned/sdk/facedetector.md

```diff
@@ -108,6 +108,14 @@ detectFaces = async imageUri => {
 // ...
 ```
 
+### `FaceDetector.isAvailableAsync()`
+
+Returns whether the Face Detection API is enabled on the current device.
+
+#### Returns
+
+Async `boolean`, indicating whether the Face Detection API is available on the current device. Currently this resolves `true` on iOS and Android only.
```

Comment:
```suggestion
Async `boolean`, indicating whether the Face Detection API is available on the current device. Currently this may resolve to `true` on iOS and Android only if the native code is linked properly.
```